rubygem-unicode/license.txt

```diff
@@ -0,0 +1 @@
+../.git/annex/objects/q6/66/SHA256E-s2504--5572bf03c3c8c4770e8153038f5f54faad9c3db9900864a7c82a19de4434d74b.txt/SHA256E-s2504--5572bf03c3c8c4770e8153038f5f54faad9c3db9900864a7c82a19de4434d74b.txt
```

Comment:
I don't think setup_sources will pickup Source1 but this is such a small, non-binary file that you can include it directly.